modules/standard/Error.chpl

```diff
@@ -42,6 +63,14 @@ proc quote_string(s:string, len:ssize_t) {
   return ret;
 }
 
+/* Halt with a useful message if there was an error. Do nothing if the error
+   argument does not indicate an error occured. The error message printed
+   when halting will describe the error passed and msg will be appended to it.
+
+   :arg error: the error object
+   :arg msg: extra information to print after the error description
+   :returns: does not return if there was an error
```

Comment:
I mean that if there was an error the function never returns, not that it doesn't return anything. You know, like how exit(1) never returns.